app/api-v2/api-doc.js

```diff
@@ -0,0 +1,138 @@
+const { PORT, API_VERSION, API_MAJOR_VERSION } = require('../env')
+
+const apiDoc = {
+  openapi: '3.0.3',
+  info: {
+    title: 'ApiService',
+    version: API_VERSION,
+  },
+  servers: [
+    {
+      url: `http://localhost:${PORT}/${API_MAJOR_VERSION}`,
+    },
+  ],
+  components: {
+    responses: {
+      NotFoundError: {
+        description: 'This resource cannot be found',
+      },
+      BadRequestError: {
+        description: 'The request is invalid',
+      },
+      UnauthorizedError: {
+        description: 'Access token is missing or invalid',
+      },
+      Error: {
+        description: 'An error occurred',
+      },
+    },
+    schemas: {
+      AuthToken: {
+        type: 'object',
+        properties: {
+          access_token: {
+            description: 'authentication token',
+            type: 'string',
+          },
+          expires_in: {
+            description: 'expiry delta time in milliseconds',
+            type: 'number',
+          },
+          token_type: {
+            description: 'type of the authentication token',
+            type: 'string',
+          },
+        },
+      },
+      Item: {
+        type: 'object',
+        properties: {
+          id: {
+            description: 'id of the item',
+            type: 'number',
+          },
+          owner: {
+            description: 'owner token of the item',
+            type: 'string',
+          },
+          creator: {
+            description: 'creator token of the item',
+            type: 'string',
+          },
+          created_at: {
+            description: 'block number the item was created at',
+            type: 'number',
+          },
+          destroyed: {
+            description: 'block number the item was destroyed',
+            type: 'number',
+          },
+          parents: {
+            description: 'parents of the item',
+            type: 'array',
+          },
+          children: {
+            description: 'children of the item',
+            type: 'array',
+          },
+          metadata: {
+            description: 'metadata of the item',
+            type: 'array',
```

Comment:
I think these array types need to be defined as
`type: 'array',
items:{}`

also to be really pernickety, metadata here only returns an array of metadata **keys** rather than all metadata. Maybe we should change the property name to `metadata_keys`

Reply:
makes sense to change the property name to `metadata_keys`

Reply:
👍